frontend/src/store/stats/reducers.ts

```diff
@@ -0,0 +1,31 @@
+// import { createReducer } from "@reduxjs/toolkit";
+// import { initialState } from "../root-schema";
+import { gamePlayed, gameWon, roundPlayed, roundWon } from "./actions";
+import { PlayerStats } from "../../models/stats.model";
+import { RootSchema } from "../root-schema";
+
+export const statsReducer =  {
```

Comment:
As mentioned above, we could get rid of these 4 actions and instead use the logic from phasespecificcenter

Reply:
I would leave it as actions because it creates cool redux history of the actions